common/src/main/scala/com/gu/media/youtube/YouTubePartnerApi.scala

```diff
@@ -165,7 +165,11 @@ trait YouTubePartnerApi { this: YouTubeAccess with Logging =>
     YoutubeRequestLogger.logRequest(YoutubeApiType.PartnerApi, YoutubeRequestType.GetVideoClaim)
     val response = request.execute()
 
-    if (response.getPageInfo.getTotalResults == 0) {
+    if(response == null || response.getPageInfo == null || response.getPageInfo.getTotalResults == null || response.getItems == null) {
```

Comment:
Should we add an inline comment against this?

Reply:
personally I think that would be noise, null checks when using Java libs are fairly self-explanatory and the log message beneath hints at this, plus if anybody is really interested they can git blame to find this PR...

Reply:
I interpreted this change as "youtube's apis cannot be trusted" more than anything else. Happy to omit the comment if git blame is enough.